### ISSUE TITLE
Results without pretrained on external dataset.

### ISSUE THREAD

**Issue by nero1342** [amazon-science/polygon-transformer#11]
Thanks for your great work!

Do you conduct experiments on your work without pretraining on Flickr30K and Visual Genome?

Thanks.

**Comment by joellliu** [comment]
Yes, we have tried that and found pretraining helps to improve the performance.

**Comment by nero1342** [comment]
Can you provide the results when training without pretraining? I think it would be fairer with others.

**Comment by joellliu** [comment]
Hi, the results without pretraining are intermediate and we don't have the result as our final setting. We use the same pretraining dataset as SeqTR.

**Comment by nero1342** [comment]
Hi, I checked the pretraining dataset and realized that Visual Genome uses large COCO images. 
You removed validation and testing images in the fine-tuning stage but not in the pre-training stage and the annotation leaking happened. 

There are some stats I have conducted: 
Total VG images:  108077
Total COCO images in VG:  51208
- [refcoco - testA] has total of 750 images, with 283 images in VG
- [refcoco - testB] has total of 750 images, with 286 images in VG
- [refcoco - val] has total of 1500 images, with 574 images in VG
- [refcoco+ - testA] has total of 750 images, with 283 images in VG
- [refcoco+ - testB] has total of 750 images, with 286 images in VG
- [refcoco+ - val] has total of 1500 images, with 574 images in VG
- [refcocog - val] has total of 1300 images, with 521 images in VG
- [refcocog - test] has total of 2600 images, with 1057 images in VG

The whole validation and testing dataset has a total of 6549 images, with 1057 images in VG.
